JSDemos/Demos/PivotGrid/RunningTotals/description.md

```diff
@@ -1 +1,3 @@
-The PivotGrid component is&nbsp;able to&nbsp;display running total values next to&nbsp;summary cells. The component can perform cross-group accumulation of&nbsp;running total values or&nbsp;accumulate them within each group separately.
\ No newline at end of file
+The PivotGrid can calculate running totals by rows or columns. To specify the direction, set a field's [runningTotal](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#runningTotal) property to *"row"* or *"column"*. This property is available only for data fields.
+
+Running totals are calculated from scratch for each [group](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#groupName) within a row or column. If you want to disable this behavior, assign **true** to the [allowCrossGroupCalculation](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#allowCrossGroupCalculation) property. In this case, the calculation continues across all groups.
```

Comment:
> If you want to disable this behavior

This phrase tells us no useful information by itself. Moreover, you actually convey useful information in the last sentence:

> In this case, the calculation continues across all groups.

```suggestion
Running totals are calculated from scratch for each [group](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#groupName) within a row or column. You can assign **true** to the [allowCrossGroupCalculation](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#allowCrossGroupCalculation) property to calculate running totals across all groups.
```